File: EmployerTerminal/Models/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EmployerTerminal.Models
{
    public class Project
    {
        public int Project_id { get; set; }
        public int Project_section_id { get; set; }
        public int Project_skills_id { get; set; }
        public string Project_name { get; set; }
        public string Project_description { get; set; }
        public string Project_image_url { get; set; }
        public string Project_url { get; set; }
        public string Project_git_url { get; set; }
    }
}

File: server/models/Project.js
const mongoose = require('mongoose');

const projectSchema = mongoose.Schema({
  project_id: { type: Number, required: true },
  project_section_id: { type: Number, ref: 'ProjectSection', required: true },
  project_name: { type: String, required: true },
  project_description: { type: String, required: true },
  project_image_url: { type: String, required: true },
  project_url: { type: String, required: true },
  project_git_url: { type: String, required: true }
},
  {collection: 'project_table'}
);

module.exports = mongoose.model('Project', projectSchema);